.github/workflows/dependencies/dependencies_hip.sh

```diff
@@ -34,7 +34,7 @@ sudo apt-get install -y --no-install-recommends \
     libnuma-dev     \
     libopenmpi-dev  \
     openmpi-bin     \
-    rocm-dev rocrand
+    rocm-dev rocrand rocprim
```

Comment:
@ax3l When you update AMReX commit in WarpX, don't forget to add rocprim to the CI for hip.